docs/reference/configuration.md

```diff
@@ -84,6 +84,7 @@ This reference uses `.` to denote the nesting of values.
 * `auth.cache.jitter` `(time duration : "3s")` - A random amount of time between 0 and this value is added to each item's TTL. This is done to avoid a large bulk of keys expiring at once and overwhelming the database.
 * `auth.encrypt.secret_key` `(string : required)` - A random (cryptographically safe) generated string that is used for encryption and HMAC signing
 * `auth.login_duration` `(time duration : "168h")` - The duration the login token is valid for
+* `auth.login_max_duration` `(time duration : "168h")` - The maximum duration user can ask for a login token
```

Comment:
please add `external_principals_enabled` as well to docs. 

(I know know why `auth.authentication_api` is not already in the docs but we do need those 2 already: 
`auth.authentication_api.endpoint` 
`auth.authentication_api.external_principals_enabled`